frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -84,7 +84,11 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
               </NavDropdown>
             </Nav>
 
-
+            <Nav className="mr-auto">
+              <NavDropdown title="Professor" id="appnavbar-professor-dropdown" data-testid="appnavbar-professor-dropdown" >
+                <NavDropdown.Item href="/professor/search" data-testid="appnavbar-professor-search">Search</NavDropdown.Item>
+              </NavDropdown>
+            </Nav>
             
```

Comment:
@Arik suggested that this be instructor instead of professor.

I tend to agree, not only because not all instructors are technically professors, but also because that's the name that's used throughout the rest of the code base and the API.   It can be confusing to have two different names for the same thing.

Having said that, if there's a good case to be made that this should be professor instead of instructor, the team can have that discussion.

I'm also wondering what users will think if they just see "Professor" or even "Instructor".    I wonder if people will think that this means "search by instructor"... or will they think it means "This is a menu for instructors only?"

Would it make more sense for this to be a menu called "Search By..." and then when you pull down the menu, you see options such as "... Instructor", "... Course Number over time", "... Locations", etc.